graphql_compiler/schema_generation/sqlalchemy/__init__.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2019-present Kensho Technologies, LLC.
+from ...schema.schema_info import SQLAlchemySchemaInfo
+from ..graphql_schema import get_graphql_schema_from_schema_graph
+from .edge_descriptors import get_join_descriptors_from_edge_descriptors, validate_edge_descriptors
+from .schema_graph_builder import get_sqlalchemy_schema_graph
+
+
+def get_sqlalchemy_schema_info_from_specified_metadata(
+    vertex_name_to_table, direct_edges, dialect, class_to_field_type_overrides=None
+):
+    """Return a SQLAlchemyInfo from the metadata.
+
+    Args:
+        vertex_name_to_table: dict, str -> SQLAlchemy Table. This dictionary is used to generate the
+                              GraphQL objects in the schema in the SQLAlchemySchemaInfo. Each
+                              SQLAlchemyTable will be represented as a GraphQL object. The GraphQL
+                              object names are the dictionary keys. The fields of the GraphQL
+                              objects will be inferred from the columns of the underlying tables.
+                              The fields will have the same name as the underlying columns and
+                              columns with unsupported types, (SQL types with no matching GraphQL
+                              type), will be ignored.
+        direct_edges: dict, str-> DirectEdgeDescriptor. Direct edges are edges that do not
+                      use a junction table, (see junction_table_edges). The traversal of a direct
```

Comment:
since you removed `junction_table_edges`, you can't talk about them. You can add them back as an argument, and immediately raise `NotImplementedError` if they are used, or update the docs not to reference them. Doing the former means fewer api changes, the latter means the api is less misleading in the meantime.

Reply:
I'll remove the reference.